server/user-bootstrap/shared-utils.js

```diff
@@ -1,6 +1,6 @@
-// These could be rewritten as `export * from`, pending resolution of Babel
```

Comment:
This is the glaring bug with `sort-imports`. It deletes valid first-docs which shouldn't exist usually 😢